pkg/provenance/sign_test.go

```diff
@@ -34,7 +34,7 @@ const (
 	// phrase. Use `gpg --export-secret-keys helm-test` to export the secret.
 	testKeyfile = "testdata/helm-test-key.secret"
 
-	// testPasswordKeyFile is a keyfile with a password.
```

Comment:
this is of course inconsequential. but please prefer distinct PRs for distinct changes (this is fixing a spelling mistake, other parts of the PR are fixing error return values)

Reply:
At first I thought that opening a new PR would increase the burden of the maintainer

Thank you for reminding, I will pay attention next time. Do I need to return to the revision here?